.github/workflows/ci.yml

```diff
@@ -28,6 +28,19 @@ jobs:
       - run: tea typecheck
       - run: tea deno lint
 
+  # test executable markdown on own repo
+  test-markdown:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: teaxyz/setup@v0
+      - run: tea .
```

Comment:
Needs to be `tea run .` or it won’t run with the current source code. Our `run` target runs `deno` with the working directory sources.

Reply:
Also you may need to rebase since I just fixed the “cannot write” error.

Reply:
But documentation says `tea .` should work  - https://github.com/teaxyz/cli#executable-markdown
```sh
$ tea .  # interprets `# Getting Started`, could also be `tea ./README.md`
```

Reply:
Rebased.

Reply:
`run` is an exe/md target in our `README`.